### ISSUE TITLE
Error while trying to start a new session on version 1.2.3

### ISSUE THREAD

**Issue by jonasneves** [gjbae1212/gossm#23]
I get the following error while trying to start a new session:
```
$ gossm-1.2.3 start
? Choose a region in AWS: us-east-1
? Choose a target in AWS: <some_instance>	(i-000000000)
[start-session] profile: env, region: us-east-1, target: i-000000000

Starting session with SessionId: <my_session_id>


SessionId: <my_session_id> :
----------ERROR-------
Encountered error while initiating handshake. Handshake timed out. Please ensure that you have the latest version of the session manager plugin.
```

The session manager plugin version I have installed in my system and for gossm are the latest:
$ session-manager-plugin --version
1.2.30.0
$ ~/.gossm/session-manager-plugin --version
1.2.30.0

Note: I am able to start a session just fine with the AWS CLI command.

**Comment by jonasneves** [comment]
I disabled encryption of the session temporarily to test and it worked. Looks like the same issue from a previous ticket I closed.

**Comment by eskp** [comment]
Seeing the same issue with a custom KMS key and the latest version of everything -  `gossm` (1.3.2), `session-manager-plugin` (1.2.54.0) and `ssm-agent` (3.0.655.0). Downgrading `gossm` to v1.0.5 I am able to connect successfully.

**Comment by satheesh1997** [comment]
I tried adding the credentials to ~/.aws/credentials file and it worked for me :)

**Comment by gjbae1212** [comment]
Mayb fix it in gossm v1.4.0.
